pkg/util/validation/validation_test.go

```diff
@@ -113,3 +116,105 @@ func TestValidateOverrideSpec(t *testing.T) {
 		}
 	}
 }
+
+func TestValidatePolicyFieldSelector(t *testing.T) {
+	desiredProvider := []string{"huaweicloud"}
+	undesiredRegion := []string{"cn-north-1"}
+	undesiredZone := []string{"foo"}
```

Comment:
```suggestion
	fakeProvider := []string{"fooCloud"}
	fakeRegion := []string{"fooRegion"}
	fakeZone := []string{"fooZone"}
```
Here we actually don't care the matching result, so the `provider/region/zone` just a literally string.

Reply:
Resolved. @RainbowMango